src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.ItemGetter.cs

```diff
@@ -0,0 +1,291 @@
+                _ => throw new ArgumentException("Unexpected interface member symbol kind")
+            };
+
+        private static string ToDisplayString(IEventSymbol symbol)
+            => symbol.Name;
+
+        private static SyntaxToken FindStartingToken(SyntaxTree tree, int position, CancellationToken cancellationToken)
+        {
+            var token = tree.FindTokenOnLeftOfPosition(position, cancellationToken);
+            return token.GetPreviousTokenIfTouchingWord(position);
+        }
+
+        private string ToDisplayString(IPropertySymbol symbol, SemanticModel semanticModel)
+        {
+            using var _ = PooledStringBuilder.GetInstance(out var builder);
+            var startToken = FindStartingToken(SyntaxTree, Position, CancellationToken);
+
+            if (symbol.IsIndexer)
+            {
+                builder.Append("this");
+            }
+            else
+            {
+                builder.Append(symbol.Name);
+            }
+
+            if (symbol.Parameters.Length > 0)
+            {
+                builder.Append('[');
+                AddParameters(symbol.Parameters, builder, semanticModel);
+                builder.Append(']');
+            }
+
+            return builder.ToString();
+        }
+
+        private string ToDisplayString(IMethodSymbol symbol, SemanticModel semanticModel)
+        {
+            using var _ = PooledStringBuilder.GetInstance(out var builder);
+            switch (symbol.MethodKind)
+            {
+                case MethodKind.Ordinary:
+                    builder.Append(symbol.Name);
+                    break;
+                case MethodKind.UserDefinedOperator:
+                case MethodKind.BuiltinOperator:
+                    AppendOperatorKeywords(symbol, builder);
+                    builder.Append(SyntaxFacts.GetText(SyntaxFacts.GetOperatorKind(symbol.MetadataName)));
+                    break;
+                case MethodKind.Conversion:
+                    AppendOperatorKeywords(symbol, builder);
+                    AddType(symbol.ReturnType, builder, semanticModel);
+                    break;
+            }
+
+            AddTypeArguments(symbol, builder);
+            builder.Append('(');
+            AddParameters(symbol.Parameters, builder, semanticModel);
+            builder.Append(')');
+            return builder.ToString();
+        }
+
+        private static void AppendOperatorKeywords(IMethodSymbol symbol, StringBuilder builder)
+        {
+            builder.Append("operator ");
+            if (SyntaxFacts.IsCheckedOperator(symbol.MetadataName))
+            {
+                builder.Append("checked ");
+            }
+        }
+
+        private void AddParameters(ImmutableArray<IParameterSymbol> parameters, StringBuilder builder, SemanticModel semanticModel)
+        {
+            builder.AppendJoinedValues(", ", parameters, (parameter, builder) =>
+            {
+                builder.Append(parameter.RefKind switch
```

Comment:
this is simpelr as `(parameter.RefKind, parameter.ScopedKind) switch => ...`

Reply:
Changed switch